web/themes/custom/votegov/templates/navigation/menu--footer.html.twig

```diff
@@ -0,0 +1,44 @@
+{#
+/**
+ * @file
+ * Vote.gov USWDS theme implementation for the footer menu.
+ *
+ * Available variables:
+ * - menu_name: The machine name of the menu.
+ * - items: A nested list of menu items. Each menu item contains:
+ *   - attributes: HTML attributes for the menu item.
+ *   - below: The menu item child items.
+ *   - title: The menu link title.
+ *   - url: The menu link url, instance of \Drupal\Core\Url
+ *   - localized_options: Menu link localized options.
+ *   - is_expanded: TRUE if the link has visible children within the current
+ *     menu tree.
+ *   - is_collapsed: TRUE if the link has children within the current menu tree
+ *     that are not currently visible.
+ *   - in_active_trail: TRUE if the link is in the active trail.
+ *
+ * @ingroup themeable
+ */
+#}
+{% import _self as menus %}
+
+{% set attributes = attributes.addClass('grid-row grid-gap') %}
+{{ menus.menu_links(items, attributes, 0) }}
+
+{% macro menu_links(items, attributes, menu_level) %}
+  {% import _self as menus %}
+
+  {% if items %}
+  {% set items_count = (items|length) / 4 %}
+  {% for item_group in items | batch(items_count) %}
+        <div class="tablet:grid-col-3">
+          <ul class="usa-list usa-list--unstyled identifier-links">
```

Comment:
We should check with Amy but breaking up a menu into several lists might be odd for screen readers. We might consider using a single list and using css `column-count` to achieve a multi-column layout.